README.md

```diff
@@ -55,3 +61,152 @@ Oscarsson, M. et al. 2019. “MXCuBE2: The Dawn of MXCuBE Collaboration.” Jour
 Gabadinho, J. et al. (2010). MxCuBE: a synchrotron beamline control environment customized for macromolecular crystallography experiments. J. Synchrotron Rad. 17, 700-707
 ```
 
+## Installation and testing
+
+We recommend using conda to create an isolated environment, conda can be downloaded [here](https://docs.conda.io/en/latest/miniconda.html#linux-installers).
+
+### 1. Create a folder that will contain both `mxcubeweb` and `mxcubecore`
+```
+mkdir mxcube
+cd mxcube
+```
+
+### 2. Clone `mxcubeweb` and `mxcubecore`
+
+```
+git clone https://github.com/mxcube/mxcubeweb.git
+git clone https://github.com/mxcube/mxcubecore.git
+```
+
+### 3. Create Conda environment `mxcubeweb`
+Create Conda environment `mxcubeweb` using the conda environment file in mxcubeweb. The installation of the dependencies in the environment takes a few minutes.
+
+The name of the environment is located at the top of file `conda-environment.yml`. It can be changed from the command line with `--name <env-name>`.
+
+```
+cd mxcubeweb
+conda env create -f conda-environment.yml
+# or (to pass a different name)
+conda env create -f conda-environment.yml --name another_name
+```
+
+### 4. Activate the environment
+
+```
+conda activate mxcubeweb
+```
+
```

Comment:
Do we oblige to use conda or we strongly recommend it, but leave it optional?

Reply:
Id say that we recommend it but one is of course free to choose another way of installing things

Reply:
In this case, maybe we say something like - We use conda for a convenience and to be independent from other projects libraries and python version. This, though, is not compulsory and one can use any other tool or configuration for installing the application and the necessary third party libraries.